metaflow/cli.py

```diff
@@ -650,7 +650,7 @@ def resume(
             )
 
     if step_to_rerun is None:
-        clone_steps = set()
+        rerun_steps = set()
```

Comment:
minor nit - could we do `steps_to_rerun` - reads a bit better

Reply:
done